pkg/kubectl/cmd/resource/get.go

```diff
@@ -327,14 +328,14 @@ func (options *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []str
 				printWithNamespace = false
 			}
 
-			var outputOpts *printers.OutputOptions
+			printOpts := *cmdutil.ExtractCmdPrintOptions(cmd, printWithNamespace)
```

Comment:
It would be better if ExtractCmdPrintOptions returned a pointer.  This is generally considered a code smell (to have to `*` a function return).